modules/optimization/include/postprocessors/VariableFunctionElementIntegral.h

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ElementIntegralPostprocessor.h"
+
+/**
+ * Integrates a function over elements
+ */
+class VariableFunctionElementIntegral : public ElementIntegralPostprocessor
+{
+public:
+  static InputParameters validParams();
+
+  VariableFunctionElementIntegral(const InputParameters & parameters);
```

Comment:
we could ditch this and just use a scaling factor on a variable integral.

Or just have 2 or 3 functors and use that

Reply:
I'm fine with removing these.  I would need some help implementing or using the objects you are talking about.